examples/README.md

```diff
@@ -1382,10 +1382,10 @@ Toolkit. It includes:
 > work. See note below.
 
 * Creation of a regional GKE cluster.
-* Creation of an autoscaling GKE node pool with `a2` machines each with 8
-  attached A100 GPUs.
-* Configuration of the cluster using the [`kubernetes-operations`] module to
-  install nvidia drivers.
+* Creation of an autoscaling GKE node pool with `g2` machines each with 1
+  attached L4 GPUs. Note: We tested this blueprint with `a2` machines as well,
+  but as capacity is hard to find we are using `g2` for example at the moment.
```

Comment:
nit: I would reword to remove the word "we". AKA, "this blueprint was tested..."